api/buildcraft/api/transport/IStripesHandler.java

```diff
@@ -17,6 +17,7 @@
 public interface IStripesHandler {
 	public static enum StripesHandlerType {
 		ITEM_USE,
+		ITEM_USE_LAST,
```

Comment:
I don't think having a publicly available ITEM_USE_LAST is a good idea. Consider making StripesHandlerDefault added to the end of the list of all stripes handlers from the handling code.